extensions/amp-consent/integrating-consent.md

```diff
@@ -59,7 +59,7 @@ window.parent.postMessage(
 );
 ```
 
-Action `'enter-fullscreen'` requests the AMP runtime to expand the iframe to fullscreen.
+Action `'enter-fullscreen'` requests the AMP runtime to expand the iframe to fullscreen. `amp-consent` will only allow the iframe enter fullscreen if it detects user interaction within iframe (e.g. clicking on the iframe).
```

Comment:
technically it's user interaction with the iframe, impossible to detect interaction within the iframe : )